autoscheduler/frontend/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.urls import path
+from django.urls import path, re_path
 
 from . import views
 
 urlpatterns = [
-    path('', views.index),
+    # match the root
+    re_path(r'^$', views.index),
+    # match all other pages
+    re_path(r'^(?:.*)/?$', views.index),
 ]
```

Comment:
Wasn't this change going to be included in #96 ?

Reply:
Yeah, I just needed it here for testing. We'll merge in #96 to master, then merge that up to frontend/master, then merge that up to frontend/schedule, then undo this change.